### PR TITLE
Issue #10548: Migrate to Inline Config Parser in InnerAssignmentCheckTest

### PR DESCRIPTION
**Pull request by shashwatj07**
#10548

### CODE REVIEW

File: src/test/resources/com/puppycrawl/tools/checkstyle/checks/coding/innerassignment/InputInnerAssignment.java [hunk 1]
```diff
@@ -19,7 +19,8 @@ void innerAssignments()
         int b;
         int c;
 
-        a = b = c = 1; // violation
+        a = b = // violation
+                c = 1; // violation
```
Comment (romani) [inline src/test/resources/com/puppycrawl/tools/checkstyle/checks/coding/innerassignment/InputInnerAssignment.java:23]: @shashwatj07 , @timurt 

can we keep single line in input ?

as violation is same can we have `a = b = c = 1;  // 2 violations`

we do not need to go crazy with parsing, we can simply support matching by 

`// violation`
`// 2 violations`

in future:
it is will be very strange  code that will require more than 2 violation. even we come to it, we can extend support to
`// 3 violation` .... `// 9 violation` . Above 9 .... I want to see real example, it should not make any sense.

**Review by romani — CHANGES_REQUESTED** [review]
item:

File: src/test/resources/com/puppycrawl/tools/checkstyle/checks/coding/innerassignment/InputInnerAssignment.java (hunk 1 above)
Comment (shashwatj07) [reply src/test/resources/com/puppycrawl/tools/checkstyle/checks/coding/innerassignment/InputInnerAssignment.java:23]: We can definitely do that. What about violation messages?

Comment (romani) [reply src/test/resources/com/puppycrawl/tools/checkstyle/checks/coding/innerassignment/InputInnerAssignment.java:23]: For now we will support only the same message for both violations

Comment (shashwatj07) [reply src/test/resources/com/puppycrawl/tools/checkstyle/checks/coding/innerassignment/InputInnerAssignment.java:23]: Okay

Comment (shashwatj07) [reply src/test/resources/com/puppycrawl/tools/checkstyle/checks/coding/innerassignment/InputInnerAssignment.java:23]: https://github.com/checkstyle/checkstyle/issues/10190

Comment (shashwatj07) [reply src/test/resources/com/puppycrawl/tools/checkstyle/checks/coding/innerassignment/InputInnerAssignment.java]: fixed

**Review by timurt — APPROVED** [review]

**Review by romani — APPROVED** [review]

**Pull request merged**